bootstrap/main.go

```diff
@@ -15,18 +15,20 @@ import (
 	"github.com/weaveworks/launcher/pkg/gcloud"
 	"github.com/weaveworks/launcher/pkg/kubectl"
 	"github.com/weaveworks/launcher/pkg/text"
+	"github.com/weaveworks/launcher/pkg/weavecloud"
 )
 
 const (
 	agentK8sURLTemplate = "{{.Scheme}}://{{.Hostname}}/k8s/agent.yaml"
 )
 
 type options struct {
-	AssumeYes bool   `short:"y" long:"assume-yes" description:"Install without user confirmation"`
-	Scheme    string `long:"scheme" description:"Weave Cloud scheme" default:"https"`
-	Hostname  string `long:"hostname" description:"Weave Cloud hostname" default:"get.weave.works"`
-	Token     string `long:"token" description:"Weave Cloud token" required:"true"`
-	GKE       bool   `long:"gke" description:"Create clusterrolebinding for GKE instances"`
+	AssumeYes  bool   `short:"y" long:"assume-yes" description:"Install without user confirmation"`
+	Scheme     string `long:"scheme" description:"Weave Cloud scheme" default:"https"`
+	Hostname   string `long:"hostname" description:"Weave Cloud launcher hostname" default:"get.weave.works"`
+	WCHostname string `long:"wc-hostname" description:"Weave Cloud hostname" default:"cloud.weave.works"`
```

Comment:
We use a `.` elsewhere, we may want to start name-spacing the arguments in a consistent way `wc.hostname`